src/main/proto/com/google/api/codegen/config.proto

```diff
@@ -274,11 +274,19 @@ message MethodConfigProto {
 
   // Specifies sets of parameter values to be used together in a
   // sample. If this is not set, no samples are output.
+  //
+  // UNDER DEVELOPMENT: Sample generation using this field is still
```

Comment:
Better wording: "Sample generation usage of this field..."

Reply:
That seems a bit _too_ compact for clarity. I tweaked it slightly differently. LMKWYT.